js_runner.go

```diff
@@ -196,7 +199,7 @@ func (runner *JSRunner) Call(inputs ...interface{}) (_ interface{}, err error) {
 			completed = make(chan struct{})
 			defer func() {
 				if caught := recover(); caught != nil {
-					if caught == ErrJSTimeout {
+					if caught == panicMsgTimeout {
```

Comment:
Thought `TestTimeout` would do it with an Otto upgrade, but it doesn't and I'm not sure why.

Just going to abandon this since we want to ditch Otto anyway and it doesn't seem worthwhile investing time into it.